lib/webpacker/configuration.rb

```diff
@@ -4,12 +4,21 @@
 
 class Webpacker::Configuration < Webpacker::FileLoader
   class << self
+    def reset
+      @defaults = nil
+      super
+    end
```

Comment:
If this is solely needed for testing, then we should just do it by hand in the tests. Not keen on having test-only methods part of the public API.